### ISSUE TITLE
Required libraries are not mentioned in the README

### ISSUE THREAD

**Issue by cpdunique** [Kalanyr/gogrepoc#27]
The current README looks unmodified compared to the source of the fork (eddie3/gogrepo) and makes no mention of the requirement for the requests or OpenSSL libraries. While requests can be installed with the obvious (```pip install requests```), it it not so obvious how to install OpenSSL (```pip install pyopenssl```), so a mention of this in the README would be most welcome.

**Comment by Kalanyr** [comment]
Yeah, this is a known issue. There's an updated readme on the dev branch (though it's fallen a bit behind the dev branch)
